package.json

```diff
@@ -60,6 +61,7 @@
     "native-watchdog": "1.0.0",
     "ng2-charts": "^1.6.0",
     "node-pty": "0.9.0-beta9",
+    "plotly.js-dist": "^1.48.3",
```

Comment:
Did you see any startup time impact from adding this dependency?  We need to be somewhat careful if this gets loaded on the startup path.

Reply:
I'm moving it to dynamic load inside the component using require (after chatting to Anthony). Verified this only loads in the module on open of a notebook with a plotly component. Therefore now has no startup impact. thanks for this feedback!